config/routes.rb

```diff
@@ -0,0 +1,65 @@
+Rails.application.routes.draw do
+
+  root 'recipefinders#index'
+
+  get 'recipefinders/index'
+
+  get 'recipefinders/show/:from' => 'recipefinders#show', as: 'recipefinders_show'
+
+  get 'recipes/show/:id' => 'recipes#show', as: 'recipes_show'
+
+  # The priority is based upon order of creation: first created -> highest priority.
```

Comment:
This needs to be cleaned up a bit, the paths are all over the place for one type of thing.  For someone looking at your routes from the outside they would have trouble figuring out which route did which thing.